lib/pure/asyncdispatch.nim

```diff
@@ -1443,7 +1444,7 @@ proc recvLine*(socket: AsyncFD): Future[string] {.async, deprecated.} =
       return
     add(result, c)
 
-proc callSoon*(cbproc: proc ()) =
+asyncfutures.callSoon = proc (cbproc: proc ()) =
```

Comment:
Good point. So this PR is currently broken. I guess I'll close it until we get a good abstract interface for dispatchers.